l2-contracts/contracts/bridge/L2AssetRouter.sol

```diff
@@ -66,30 +62,22 @@ contract L2SharedBridge is IL2SharedBridge, ILegacyL2SharedBridge, Initializable
 
     /// @dev Contract is expected to be used as proxy implementation.
     /// @dev Disable the initialization to prevent Parity hack.
-    constructor(uint256 _eraChainId, uint256 _l1ChainId) {
+    constructor(uint256 _eraChainId, uint256 _l1ChainId, address _l1SharedBridge, address _l1Bridge) {
         ERA_CHAIN_ID = _eraChainId;
         L1_CHAIN_ID = _l1ChainId;
+        _initialize(_l1SharedBridge, _l1Bridge);
```

Comment:
I don't think this is needed. Esp since we'll disable initializers anyway

Reply:
We need to set the values, so I will just remove the initializer